### PR TITLE
[FEAT] Added custom share text option.

### PR DESCRIPTION
**Pull request by renesansz**
Added option for customizing share text.

### CODE REVIEW

File: src/android/io/branch/BranchSDK.java [hunk 1]
```diff
@@ -466,12 +466,12 @@ private void createBranchUniversalObject(JSONObject options, CallbackContext cal
...
      * @param controlParams A {@link JSONObject} value to set the URL control parameters.
      */
-    private void showShareSheet(int instanceIdx, JSONObject options, JSONObject controlParams) throws JSONException
+    private void showShareSheet(int instanceIdx, JSONObject options, JSONObject controlParams, String shareText) throws JSONException
     {
 
         Log.d(LCAT, "start showShareSheet()");
 
-        ShareSheetStyle shareSheetStyle = new ShareSheetStyle(this.activity, "Check this out!", "This stuff is awesome: ")
+        ShareSheetStyle shareSheetStyle = new ShareSheetStyle(this.activity, "Check this out!", shareText)
```
Comment (aaustin) [inline src/android/io/branch/BranchSDK.java]: I think we should expose the title as well instead of hard coding it to "Check this out!"

**Comment by aaustin** [comment]
Can you add this argument to iOS as well?

**Comment by renesansz** [comment]
Hey @aaustin ,
This is already have been implemented in iOS, just have to let the javascript accept the custom text argument.

**Comment by aaustin** [comment]
Sounds good. See my other note though?

**Comment by renesansz** [comment]
Yes, however I also consulted @jestoniyap in regards to the iOS.
He said that there's no implementation for iOS in order to modify the share sheet title.
That's why I disregarded this at the moment.

File: src/android/io/branch/BranchSDK.java (hunk 1 above)
Comment (renesansz) [inline src/android/io/branch/BranchSDK.java]: Yes, I supposed, however iOS doesn't have a functionality to modify the share title text according to @jestoniyap . So atm I disregarded the modification of the title, though I can still add the functionality for Android but not on iOS. If it's okay with you.

**Comment by aaustin** [comment]
Ok sounds good. Thanks. Can you fix the merge conflicts first?

**Comment by renesansz** [comment]
@aaustin fixed merge conflict.

**Comment by aaustin** [comment]
:+1:

**Pull request merged**